tests/generate_packageTest.py

```diff
@@ -0,0 +1,24 @@
+import os
+import unittest
+from bears.generate_package import touch, create_file_from_template
+
+
+class touchTest(unittest.TestCase):
+
+    def test_file_doesnt_exist(self):
+        touch('TestFile.py')
+        self.assertTrue(os.path.exists('TestFile.py'))
```

Comment:
i will, but my function does not even care if the file doesn't exist :p it does stuff only if it does exist.